doc/user_manual/postprocessor.tex

```diff
@@ -1800,10 +1801,62 @@ \subsubsection{RavenOutput}
 \end{lstlisting}
 The resulting PointSet has \emph{time} as an input and \emph{first} as an output.
 
+%%%%%%%%%%%%%% MCSImporter PP %%%%%%%%%%%%%%%%%%%
+
+\subsubsection{ParetoFrontier}
+\label{ParetoFrontierPP}
+The \textbf{ParetoFrontier} post-processor is designed to identify the points lying on the Pareto Frontier in a cost-value space.
+This post-processor receives as input a dataObject (a PointSet only) which contains all data points in the cost-value space and it 
+returns the subset of points lying in the Pareto Frontier as a PointSet.
+
+It is here assumed that each data point of the input PointSet is a realization of the system under consideration for a 
+specific configuration to which corresponds a cost and a value.
+
+%
+\ppType{ParetoFrontier}{ParetoFrontier}
+%
+\begin{itemize}
+  \item  \xmlNode{costID},\xmlDesc{string, required parameter}, ID of the input PointSet variable that is considered the cost variable 
+  \item  \xmlNode{valueID},\xmlDesc{string, required parameter}, ID of the input PointSet variable that is considered the value variable
+\end{itemize}
+
+The following is an example where a set of realizations (the ``candidates'' PointSet) has been generated by changing two parameters 
+(var1 and var2) which produced two output variables (cost and value).
+The \textbf{ParetoFrontier} post-processor takes the ``candidates'' PointSet and populates a Point similar in structure 
+(the ``paretoPoints'' PointSet).
+
+\textbf{Example:}
+\begin{lstlisting}[style=XML,morekeywords={anAttribute},caption=MCS Importer input example (no expand)., label=lst:MCS_PP_InputExample]
```

Comment:
Change the contents for caption and label, currently they are pointing to MCS Importer.